subtests/docker_cli/systemd/docker-build.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=docker-autotest systemd build image subtest. If you see this file on a live system it means something has failed very badly
+
+[Service]
+Type=oneshot
+Environment=NAME=rhel7_p4321
+Environment=OPTIONS=--force-rm
+Environment=DOCKERFILEDIR=/usr/local/autotest/client/tests/docker/subtests/docker_cli/systemd
```

Comment:
Same as above, only much more so. Make it very obvious that this is a garbage value, eg `DOCKERFILEDIR=this-will-be-replaced-by-the-test`. And try to make a habit of doing so for values that will be replaced.

The reason is that `/usr/local/autotest/blah/blah` looks "real". It looks like an actual value, and my first thought on reading it was hey, wait, no, it should be `/var/lib/autotest`, and besides there's too much duplication here, and I spent a few minutes figuring out that the value is thrown away.

When you have strings that are going to be replaced, make it obvious that they are dummy data. If you want them to look like an example, maybe use /this/is/an/example or some other convention. But don't use values that resemble real data, because they'll trick your maintenance programmer.